.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+        node_version: [12, 13]
```

Comment:
Note that I don't include node 10, as it is no longer LTS.
LTS now is 12, and 13 is the next

Reply:
Node has multiple LTS versions at once. Right now that's 8, 10 and 12, but 8 is EOL at the end of this year. So currently, we support Node 10 and 12. We don't test against non-LTS releases typically.

Reply:
Oh, I'll update this accordingly!